components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -252,6 +252,17 @@ protected String getScope(String scope, Map<String, String> authenticatorPropert
         return scope;
     }
 
+    /**
+     * Get Scopes.
```

Comment:
Would be good to have a better description. Hard to understand whats happening through the description or the implementation

Reply:
Fixed with https://github.com/wso2-extensions/identity-outbound-auth-oidc/pull/143